### PR TITLE
Update berkeley configs

### PR DESCRIPTION
**Pull request by aclum**
This PR updates several components needed for the berkeley rollout including:
- updating nmdc-schema to a berkeley candidate release, pinned rc16 until release packaging issues are resolved
- updating code and config files with new Database slot names, WorkflowExecution class names, and enumeration values
- updates needed to get tests to pass (test_sched.py not expected to pass still based on work in this PR)

### CODE REVIEW

File: tests/test_imports.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def test_activity_mapper(gold_mapper):
     gold_mapper.activity_mapper()
     gold_mapper.post_nmdc_database_object()
     db = gold_mapper.get_database_object_dump()
-    assert len(db.metagenome_annotation_activity_set) == 1
-    assert len(db.mags_activity_set) == 1
+    #TODO I think this assert statement should be 2 ANC
```
Comment (aclum) [inline tests/test_imports.py:43]: @mbthornton-lbl please double check correct value for len(db.workflow_execution_set) assertion

**Comment by ssarrafan** [comment]
@scanon @mbthornton-lbl can you review this so we can close the associated ticket by tomorrow?

**Comment by mbthornton-lbl** [comment]
@aclum @scanon  I think I have addressed the causes of the various "File Not Found" issues that are causing failing tests.  The changes are in the big #195 PR

I think the most straightforward way to incorporate these updates to get the tests running here would be:
- review and land #195 
- merge main into #219

**Comment by eecavanna** [comment]
This PR came up during today's Berkeley Schema Roll Out squad meeting. We don't know "whose court the ball is in."

**Review by mbthornton-lbl — APPROVED** [review]

**Pull request merged**